compass-js-frontend/src/main/webapp/css/layout-default.scss

```diff
@@ -151,6 +151,52 @@
     padding-bottom:	0.35ex; /* to 'vertically center' text inside text-span */
 }
 
+@mixin toggler-fontawesome($content) {
```

Comment:
This is the only non-fancytree change in this branch, it also adds little arrows to the ui-layout drawers.